### PR TITLE
0.2 release fixes

### PR DESCRIPTION
**Pull request by dpc**

### CODE REVIEW

File: client/Cargo.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ name = "bitcoincore_rpc"
 path = "src/lib.rs"
 
 [dependencies]
-bitcoincore-rpc-json = "0.2"
+bitcoincore-rpc-json = { version = "0.2", path = "../json" }
```
Comment (stevenroose) [inline client/Cargo.toml]: This doesn't work for releasing.

File: .travis.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ rust:
   - stable
   - beta
   - nightly
-  - 1.21.0 # first cargo that supports [patch]
+  - 1.31.0
```
Comment (stevenroose) [inline .travis.yml]: Why so high? I think we can be compatible with lower versions.

Comment (dpc) [reply .travis.yml]: That's what I got when looking up when const_fn was stabilized (that was error on travis). I might be wrong though.

File: client/Cargo.toml (hunk 1 above)
Comment (dpc) [reply client/Cargo.toml]: It should. I use it all over the place. Eg. https://github.com/dpc/crev/blob/609888cedd9e9484d279113467a87d758470aa57/cargo-crev/Cargo.toml#L24

**Comment by dpc** [comment]
Rebased.

File: client/Cargo.toml (hunk 1 above)
Comment (stevenroose) [reply client/Cargo.toml]: So how do you release? Last time I tried to `cargo publish` a crate with a `path = ` dependency, I got a message telling me that path dependencies are not allowed or something.

Comment (dpc) [reply client/Cargo.toml]: It just works. Always worked. :) You first release the `json`, then you go to the client and release that. `path` is ignored when doing `cargo publish`, AFAIK.

**Comment by stevenroose** [comment]
Will look at it tomorrow! Cool!

**Comment by dpc** [comment]
ship it ?

**Pull request merged**